### ISSUE TITLE
[a11y]: cds-tooltip trigger reports false color-contrast violation (background color cannot be determined due to overlapping element) in Storybook a11y panel

### ISSUE THREAD

**Issue by rsrduarteEG** [carbon-design-system/carbon#20676]
### Package

@carbon/web-components

### Package version

v2.30.0

### Automated testing tool and ruleset

@storybook/addon-a11y pluggin

### Description

In Storybook (with @storybook/addon-a11y), any text/button used as the first child trigger of cds-tooltip plus a following cds-tooltip-content reports:
"Elements must meet minimum color contrast ratio thresholds - Element's background color could not be determined because it is overlapped by another element."
This appears to be a false positive caused by the tooltip’s internal wrapping/overlay (positioning layer, pseudo-elements, or an absolutely positioned element) which makes Axe think the trigger text is overlapped, so it cannot compute the real background.
This happens even when using the first tooltip example from the Storybook documentation. Is there any required markup, attribute, or styling missing that would prevent the color contrast false positive?

### WCAG 2.1 Violation

Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds

### Reproduction/example

https://stackblitz.com/edit/github-7n6mmfou?file=index.html

### Steps to reproduce

1. Start Storybook with @storybook/addon-a11y enabled.
2. Use the reproduction/examples in a story
3. Open the Accessibility panel.
4. Observe color-contrast issue on the DIV or BUTTON containing “LABEL”.

<img width="478" height="560" alt="Image" src="https://github.com/user-attachments/assets/a981a200-56cc-4d76-b3c9-9b56ec3f2e12" />

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
